doc/api/cli.md

```diff
@@ -816,6 +816,25 @@ added: v12.0.0
 Set default [`tls.DEFAULT_MIN_VERSION`][] to 'TLSv1.3'. Use to disable support
 for TLSv1.2, which is not as secure as TLSv1.3.
 
+### `--trace-atomics-wait`
+<!-- YAML
+added: REPLACEME
+-->
+
+Print short summaries of calls to `Atomics.wait()` to stderr.
+The output could look like this:
+
+```text
+[Thread 0] Atomics.wait(0x55d134fe4290 + 0, 1, inf) started
+[Thread 0] Atomics.wait(0x55d134fe4290 + 0, 1, inf) did not wait because the values mismatched
+[Thread 0] Atomics.wait(0x55d134fe4290 + 0, 0, 10) started
+[Thread 0] Atomics.wait(0x55d134fe4290 + 0, 0, 10) timed out
+[Thread 0] Atomics.wait(0x55d134fe4290 + 4, 0, inf) started
+[Thread 1] Atomics.wait(0x55d134fe4290 + 4, -1, inf) started
+[Thread 0] Atomics.wait(0x55d134fe4290 + 4, 0, inf) was woken up by another thread
+[Thread 1] Atomics.wait(0x55d134fe4290 + 4, -1, inf) was woken up by another thread
```

Comment:
Typically `--trace-*` flags also emit the stack trace so we know where those are happening. Is that possible here or is it too much of a performance hit?

Reply:
> Is that possible here or is it too much of a performance hit?

I think that would come with a significant performance hit, yes. is already coming with one, as @devsnek made me realize in the comment above.

The stack traces might also be not quite as useful here as for other features (although in an ideal world they would of course be nice-to-have), because the number of `Atomics.wait()` lines in any given program should be somewhat limited. We could add this as an additional optional feature though, if you think that that is worth it?

Reply:
Let's hold off and see how far we get with this as is. We can always add the stack traces later. That said, we should likely document the discrepancy.

Reply:
Would be helpful in the doc to add a bit of explanation about the output format.

Reply:
Yeah, I’ve added explanations for the individual fields here :+1: